File: 8KYU/Powers of 2/solution.cs
using System;
using System.Numerics;
using System.Collections.Generic;
public class Kata
{
  public static BigInteger[] PowersOfTwo(int n)
  {
    Console.WriteLine(n);
    var result = new List<BigInteger>();
    
    for(int i = 0; i <= n; i++)
    {
      result.Add(BigInteger.Pow(2, i));
    }
    
    return result.ToArray();
  }
}

File: 8KYU/Powers of 2/solution.go
package kata

import (
	"math"
)

func PowersOfTwo(n int) []uint64 {
	var res []uint64

	for i := 0; i <= n; i++ {
		res = append(res, uint64(math.Pow(2, float64(i))))
	}
	return res
}
